pkg/mimirtool/commands/analyse.go

```diff
@@ -67,6 +67,8 @@ func (cmd *AnalyzeCommand) Register(app *kingpin.Application, envVars EnvVarName
 	grafanaAnalyzeCmd.Flag("output", "The path for the output file").
 		Default("metrics-in-grafana.json").
 		StringVar(&gaCmd.outputFile)
+	grafanaAnalyzeCmd.Flag("folder-titles", "Limit dashboards analysis for unused metrics based on their exact folder title. when repeated any of the matching folder will be analyzed.").
```

Comment:
Also maybe make the flag singular (`folder-title`) because the argument is a single title. Otherwise, you can make the flag be comma-separated values (`--folder-titles General,Ops,Some-Other-Folder`). We have CSVs for other values in mimir itself ([memcached addresses for example](https://github.com/grafana/dskit/blob/a68e8d80adaac5f81a591e97af9cc13e519d1f0f/cache/memcached_client.go#L71)) and I think I'm leaning more to that than repeating flags. But I'd maybe leave others to pitch in with opinions

Reply:
As folder title may contain any character I believe it can be not trivial in some cases to know what to escape or not.
In my experience, I've seen dot, dash, underscore emoji as folder titles.
![Capture d’écran 2023-05-12 à 17 14 27](https://github.com/grafana/mimir/assets/12611310/2dc16dc2-df2a-49b5-b6ce-acedd264f389)

Reply:
makes sense. Thanks for explaining!